modules/Administration/Diagnostic.tpl

```diff
@@ -37,93 +37,128 @@
  * reasonably feasible for technical reasons, the Appropriate Legal Notices must
  * display the words "Powered by SugarCRM" and "Supercharged by SuiteCRM".
  */
+*}
 
-/**
+<form name="Diagnostic" method="POST" action="index.php">
+    <h4>Quick Report</h4><br/>
+    <input type="hidden" name="module" value="Administration">
+    <input type="hidden" name="action" value="DiagnosticRun">
+    <!-- Keep the following report output in such a way that it copy-pastes well from browser into
+     a simple text field, such as a forum post. Ensure the whitespace between HTML tags is placed where it should. -->
+    <table cellpadding="0" cellspacing="0" border="0" width="100%" class="edit view">
+        {foreach from=$QuickReport item=section key=sectionname}
+            <tr><td style='min-width: 14em;'><br/><h5><b>{$sectionname}</b></h5><br/></td></tr>
+            {foreach from=$section item=label key=key}<tr><td>{$key}:</td><td>{$label}</td></tr>{/foreach}
+        {/foreach}
+        <tr><td>&nbsp;</td></tr>
+        <tr>
+            <td>
+                <input title="DiagnosticQuickViewPhpInfo" class="button"
+                       onclick="this.form.action.value='DiagnosticQuickViewPhpInfo';" type="submit" name="button"
+                       value="{$MOD.LBL_DIAGNOSTIC_VIEWFULLPHPINFO}">
+            </td>
+        </tr>
+        <tr><td>&nbsp;</td></tr>
+    </table>
 
- */
-*}
+</form>
 
 <form name="Diagnostic" method="POST" action="index.php">
-<input type="hidden" name="module" value="Administration">
-<input type="hidden" name="action" value="DiagnosticRun">
- 
-<table cellpadding="0" cellspacing="0" border="0" width="100%" class="actionsContainer">
-	<tr>
-	<td>
-	<input title="{$MOD.LBL_DIAG_EXECUTE_BUTTON}" class="button" onclick="this.form.action.value='DiagnosticRun';" type="submit" name="button" value="  {$MOD.LBL_DIAG_EXECUTE_BUTTON}  " >
-	<input title="{$MOD.LBL_DIAG_CANCEL_BUTTON}" class="button" onclick="this.form.action.value='index'; this.form.module.value='Administration'; " type="submit" name="button" value="  {$MOD.LBL_DIAG_CANCEL_BUTTON}  "></td>
+    <h4>Download Diagnostic Files</h4><br/>
```

Comment:
@pgorod 
Could you pls add corresponding label for 'Download Diagnostic Files' ?

Reply:
Done, thanks. I re-used a previously existing label.